sw/device/tests/crypto/cryptotest/json/ecdsa_commands.h

```diff
@@ -0,0 +1,63 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_ECDSA_COMMANDS_H_
+#define OPENTITAN_SW_DEVICE_TESTS_CRYPTO_CRYPTOTEST_JSON_ECDSA_COMMANDS_H_
+#include "sw/device/lib/ujson/ujson_derive.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ECDSA_CMD_MAX_MESSAGE_BYTES 64
+#define ECDSA_CMD_MAX_SIGNATURE_SCALAR_BYTES 64
+#define ECDSA_CMD_MAX_COORDINATE_BYTES 64
+
+// clang-format off
+
+#define ECDSA_OPERATION(_, value) \
+    value(_, Verify)
```

Comment:
It would be nice to document what is expected to follow the `Verify` device and what does the device answer. For example:
```rust
// Following a Verify command, the host is expected to send a `CryptotestEcdsaMessage`, a `CryptotestEcdsaSignature` to verify, two `CryptotestEcdsaCoordinate` (I assume the public key). The device will respond with a `CryptotestEcdsaVerifyOutput`.
```